build.sh

```diff
@@ -418,6 +425,12 @@ if [[ ${#_VERBOSE} > 0 ]]; then
     echo ""
 fi
 
+if [[ $HAS_LTTNG == 1 ]]; then
+    python tools/lttng.py --man `pwd`/manifests/Microsoft-Scripting-Chakra-Instrumentation.man --intermediate `pwd`/out/intermediate
+    mkdir -p `pwd`/out/lttng
+    diff -q `pwd`/out/intermediate/lttng/jscriptEtw.h `pwd`/out/lttng/jscriptEtw.h || cp `pwd`/out/intermediate/lttng/* `pwd`/out/lttng/
```

Comment:
Why don't you create `jscriptEtw.h` already and put it under PlatformAgnostic/Linux ? What's the gain of doing this here dynamically? 

++ This approach doesn't comply with build.sh `--target-path` folder selection.

Reply:
`jscriptEtw.h` is one of the generated files, made from the ETW manifest. We could check it in to source control, but then it would get out of date when changes are made to the ETW manifest and that would cause issues. The diff check here is to try to improve incremental builds; if `jscriptEtw.h` is the same then the manifest hasn't changed, and so (barring changes to the generation script) all the generated files should be the same, so no need to dirty them which would cause most files to be recompiled

Reply:
Regarding `--target-path` I wasn't aware of that; how would you recommend addressing that? It looks like currently the target path is contained completely within `build.sh`, and so I wouldn't be able to have the `CMakeLists.txt` reference the generated files without passing in their specific location.